### PR TITLE
Logo added to the readme file.

### PR DESCRIPTION
**Pull request by guidoguy22**
The official Bitcoin Core logo added to the readme document.

### CODE REVIEW

**Comment by laanwj** [comment]
I don't think we'd want to link an image from an external website here. If you really want to add an image (not sure it's really worth it) it has to be one in the repository itself.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![Bitcoin Core](https://bitcoincore.org/assets/images/bitcoin_core_logo.png)
```
Comment (promag) [inline README.md:1]:
```
![Bitcoin Core](doc/bitcoin_logo_doxygen.png?raw=true "Bitcoin Core")
```

**Review by promag — COMMENTED** [review]
No strong opinion.

File: README.md (hunk 1 above)
Comment (OliverPetrovic) [reply README.md]: I added the small version hosted in the repo.

Thanks!

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Bitcoin Core integration/staging tree
```
Comment (promag) [inline README.md:1]: Screenshot 2020-12-04 at 19 49 30

IMHO not worth it.

**Comment by fanquake** [comment]
I agree with the others. Let's leave the readme as is.

**Pull request closed without merging**